modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminManagementToolbarDisplayContext.java

```diff
@@ -261,6 +261,83 @@ public CreationMenu getCreationMenu() throws PortalException {
 		return creationMenu;
 	}
 
+	public List<DropdownItem> getEmptyStateActionDropdownItems() {
+		long kbFolderClassNameId = PortalUtil.getClassNameId(
```

Comment:
@boton If you add here:
```
if (Validator.isNotNull(_getKeywords()) {
    return null;
}
```
it should do what you expect. I'm getting some weird errors when doing ant all, so I cannot test it myself at the moment.